frontend/src/views/toolbox/clam/status/index.vue

```diff
@@ -65,7 +69,7 @@
                         <el-button type="primary" link @click="onOperate('FreshClam', 'restart')">
                             {{ $t('app.restart') }}
                         </el-button>
-                    </span>
+                    </div>
                 </div>
             </el-card>
         </div>
```

Comment:
I'm sorry, but you have not provided the Python code that needs to be checked. Please send the relevant Python code so I can review it.